cluster/examples/kubernetes/ceph/ec-storageclass.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: ceph.rook.io/v1beta1
+kind: Pool
+metadata:
+  name: ec-metadata-pool
```

Comment:
should we use the prefix `replicated` instead of `ec` on the name of this pool? It seems confusing like this.